app/test_split_integer.py

```diff
@@ -2,20 +2,34 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    result = split_integer(8, 1)
+    assert sum(result) == 8
```

Comment:
According to the checklist, avoid creating unnecessary variables for simple tests. You can directly assert the function call: `assert sum(split_integer(8, 1)) == 8`